e2e/rbd.go

```diff
@@ -409,6 +425,15 @@ var _ = Describe("RBD", func() {
 		if err != nil {
 			framework.Failf("failed to delete node label: %v", err)
 		}
+		// Remove the CRUSH Location labels
```

Comment:
same here as well, lets update deleteNodeLabel to take multiple labels (if possible)